components/dashboard/SentimentTracker.js

```diff
@@ -8,7 +8,6 @@ import React from 'react';
 import Typography from '@material-ui/core/Typography';
 
 import { useAuth } from '../Auth';
-import EmojiCircle from './EmojiCircle';
```

Comment:
Did a quick grep across the codebase and I couldn't see this file used anywhere else, could we maybe delete the file `components/dashboard/EmojiCircle.js` ?

Reply:
deleted. thanks!